src/rgw/rgw_op.cc

```diff
@@ -5031,6 +5031,12 @@ void RGWOptionsCORS::execute()
 
 int RGWGetRequestPayment::verify_permission()
 {
+  if (s->iam_policy &&
+      s->iam_policy->eval(s->env, *s->auth.identity,
+			  rgw::IAM::s3GetBucketRequestPayment,
+			  ARN(s->bucket)) != Effect::Allow) {
```

Comment:
For a better coding style, I think we can keep `if (...)` consistent between `RGWGetRequestPayment::verify_permission()` and `RGWSetRequestPayment::verify_permission()`.

Of course, now it is also all right.